.github/workflows/integration.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.8'
+          python-version: '3.10'
```

Comment:
I can get behind running the tests with `3.10` or even `3.12`.  What did you run into with 3.8 that didn't work?